src/charm.py

```diff
@@ -1015,6 +1022,11 @@ def _create_pgdata(self, container: Container):
             "/var/lib/postgresql/temp",
         ]).wait()
 
+    def _on_start(self, _) -> None:
+        # Make sure the CA bubdle file exists
+        # Bundle is not secret
+        Path(f"/tmp/{TLS_CA_BUNDLE_FILE}").touch()  # noqa: S108
```

Comment:
`requests` lib expects that the verify bundle file exists, so make sure that an empty file is there from the get go.